collection/rancher/v2.x/logs-collector/README.md

```diff
@@ -14,7 +14,7 @@ The script will create a .tar.gz log collection in /tmp by default, all flags ar
 
 ```
 Rancher 2.x logs-collector
-  Usage: rancher2_logs_collector.sh [ -d <directory> -s <days> -r <container runtime> -p -f ]
+  Usage: rancher2_logs_collector.sh [ -d <directory> -s <days> -r <container runtime> -p -f -u -c <case number> -l <location> ]
```

Comment:
What about providing an option to select an already generated support bundle and upload it?

Reply:
Oh, I hadn't thought of that.